extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -37,10 +37,42 @@
     @ConfigItem
     public List<String> topics;
 
+    /**
+     * The schema registry key.
+     * e.g. to diff between different registry impls / instances
+     * as they have this registry url under different property key.
+     */
+    @ConfigItem(name = "schema.registry.key", defaultValue = "schema.registry.url")
+    public String schemaRegistryKey;
+
+    /**
+     * The schema registry url.
+     */
+    @ConfigItem(name = "schema.registry.url")
```

Comment:
In Quarkus config, we use a dot to separate sections and a dash to separate words. It's a general comment about the config key of this PR.